### ISSUE TITLE
What is "from .version import __version__" in __init__.py? 

### ISSUE THREAD

**Issue by MateoLostanlen** [pyronear/pyro-vision#49]
## 🐛 Bug

The import "from .version import __version__" in __init__.py? generate the following error :

ModuleNotFoundError: No module named 'pyronear.version'

I use to comment it, it seems to make no difference. Can we remove it ?

**Comment by frgfm** [comment]
It is not required for dev, but only for common installation!
In https://github.com/frgfm/PyroNear/blob/master/setup.py#L32-L35, you can see that installing the lib will create this file with the correct package version, so that you can have the following behaviour:
```
import pyronear
print(pyronear.__version__)
```
yields 
```
X.Y.Z
```
like in most python packages.
Let me know if that clears things up!

**Comment by frgfm** [comment]
Closing for now. Feel free to reopen if you think this is actually an unwanted behaviour!

**Comment by x0s** [comment]
Since version number is manually written and only once, why not writing directly in an external file 
`__version__ = 0.2.0a0` in a `version.py`
and execute this file in setup like [it is advised in official doc](https://packaging.python.org/guides/single-sourcing-package-version/)

This way, we should be able to avoid import race issue in setup.py and avoid import error when trying to access `__version__` in dev mode.

What do you think?

**Comment by x0s** [comment]
Besides, I reopen this issue because no solution has been provided to cope with the raised exception.
For instance, every time tests are launched (so in dev mode):

Here to reproduce the behaviour:
` $ python -m unittest -v test_datasets_wildfire_split.WildFireDatasetTester`
yields

```
test_datasets_wildfire_split (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: test_datasets_wildfire_split (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_datasets_wildfire_split
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/.../PyroNear/test/test_datasets_wildfire_split.py", line 13, in <module>
    from pyronear.datasets.wildfire import (WildFireDataset,
  File "/.../PyroNear/pyronear/__init__.py", line 1, in <module>
    from .version import __version__
ModuleNotFoundError: No module named 'pyronear.version'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

One side-effect of manually commenting, is that one can [inadvertently _push_ that trick on repository](https://github.com/frgfm/PyroNear/pull/48/commits/e622b6562ffb2b71165d458e62bbdbfe0d8a5a7a#diff-10db0c79bdfa381f0d6b75329fdb19c4)

It happens because, this never-to-be-committed change persist the working directory. So it is very easy to unintentionally commit it

![Screenshot from 2020-04-01 14-53-54](https://user-images.githubusercontent.com/6331909/78139350-a9880900-7428-11ea-8600-7ef2975e392a.png)

**Comment by Akilditu** [comment]
Same issue here, comment remains the only solution for dev ?

**Comment by frgfm** [comment]
> Same issue here, comment remains the only solution for dev ?

@Akilditu Just made a PR with fix proposal here #57
